src/investarena/redux/reducers/modalsReducer.js

```diff
@@ -30,3 +32,5 @@ export default function(state = initialState, action) {
       return state;
   }
 }
+
+export const activeModal = state => state.activeModal;
```

Comment:
you can replace this selector with smth like this:
`export const isActiveModal = state => Boolean(state.modals.length);`